src/components/ActionsMenu/ActionsMenu.driver.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { StylableUnidriverUtil, UniDriver } from 'wix-ui-test-utils/unidriver';
+import style from './ActionsMenu.st.css';
+
+export interface ActionsMenuDriver extends BaseUniDriver {
+  isMobile(): Promise<boolean>;
+}
+
+export const actionsMenuDriverFactory = (
+  base: UniDriver,
+): ActionsMenuDriver => {
+  const stylableUtil = new StylableUnidriverUtil(style);
```

Comment:
We usually avoid using `StylableUnidriverUtil` because it causes problems in e2e tests
You can add a `data-` attribute to your component and verify using this attribute

Reply:
Fixed.
It was basically code from generator, maybe we need to change it also?